core/chain_manager.go

```diff
@@ -48,6 +48,9 @@ var (
 )
 
 const (
+	headerCacheLimit    = 512
+	bodyCacheLimit      = 256
+	tdCacheLimit        = 1024
```

Comment:
TD is typically used together with the header, would it make sense to have the same cache number for both?